src/matcha_ml/cli/cli.py

```diff
@@ -86,7 +86,7 @@ def get(
         Exit: Exit if resource type or property does not exist in matcha.state.
     """
     try:
-        resources = core.get(resource_name, property_name)
+        resources = core.get(resource_name, property_name).to_dict()
```

Comment:
If we did add `__dict__` to `MatchaState` (see other comments), is it cleaner to access this as `resources = dict(...)`?

Reply:
It's cleaner in that there's less code but it's also less specified - I'd prefer an explicit call to a `to_dict` (or `asdict`) function